### ISSUE TITLE
Feature: Default when not found

### ISSUE THREAD

**Issue by billneff79** [developit/dlv#2]
I've found us doing this pattern a lot when using dlv:

```
let foo = dlv(bar, 'a.b.c');
if(typeof foo === 'undefined') foo = defaultFoo;
```

It might be useful to allow dlv to support a third argument that is a default return value if the key is not found

**Comment by developit** [comment]
Sounds like a nice addition.

**Comment by developit** [comment]
Fix merged and released as [1.1.0](https://github.com/developit/dlv/releases/tag/1.1.0).
